packages/superset-ui-core/src/query/types/Query.ts

```diff
@@ -41,11 +41,24 @@ export type QueryObjectExtras = Partial<{
   where?: string;
 }>;
 
+export type QueryObjectAppliedTimeExtraKey =
```

Comment:
Can we rename this to something more generic such as `TimeColumnConfigKey`? Also, maybe put this into `Time.ts` ?